src/policy/rbf.cpp

```diff
@@ -181,3 +181,36 @@ std::optional<std::string> PaysForRBF(CAmount original_fees,
     }
     return std::nullopt;
 }
+
+std::optional<std::string> CheckMinerScores(CAmount replacement_fees,
+                                            int64_t replacement_vsize,
+                                            const CTxMemPool::setEntries& direct_conflicts,
+                                            const CTxMemPool::setEntries& original_transactions)
+{
+        // Note that this assumes no in-mempool ancestors
+        const CFeeRate replacement_miner_score(replacement_fees, replacement_vsize);
+
+        for (const auto& entry : direct_conflicts) {
+            const bool conflict_is_v3{entry->GetSharedTx()->nVersion == 3};
```

Comment:
this can't be hit yet; post-v3 it can. I can remove this, or leave a note. It makes some package rbfs more useful